src/h5web/nexus-visualizer/utils.ts

```diff
@@ -0,0 +1,30 @@
+import type { HDF5Dataset, HDF5Group, HDF5Value } from '../providers/models';
+import type { Vis } from '../dataset-visualizer/models';
+import {
+  NXDataAttribute,
+  NXDataInterpretation,
+  INTERPRETATION_VIS,
+} from './models';
+
+export function getAttributeValue(
+  entity: HDF5Dataset | HDF5Group,
+  attributeName: NXDataAttribute
+): HDF5Value {
+  if (!entity.attributes) {
+    return undefined;
+  }
+
+  return entity.attributes.find((attr) => attr.name === attributeName)?.value;
```

Comment:
You didn't like `entity.attributes?.find(...)` ? Meh, doesn't matter 👍

Reply:
As much as I find `?.` practical when coding, I feel it can be hard to catch when reading the code. So I prefer to have something explicit instead whenever I can :wink: